scipy/interpolate/interpnd.pyx

```diff
@@ -873,7 +873,7 @@ class CloughTocher2DInterpolator(NDInterpolatorBase):
     continuously differentiable.
 
     The gradients of the interpolant are chosen so that the curvature
-    of the interpolating surface is approximatively minimized. The
+    of the interpolating surface is approximately minimized. The
```

Comment:
This [shouldn't](https://www.thefreedictionary.com/approximatively) be changed.

Reply:
Why? For some reason, I prefer `approximatively` myself, but the word is missing from all common dictionaries:
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=approximatively)
* [Cambridge](https://dictionary.cambridge.org/fr/spellcheck/anglais/?q=approximatively)
* [Collins](https://www.collinsdictionary.com/spellcheck/english?q=approximatively)
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=approximatively)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/approximatively)

I don't mean to say  `approximatively` is not an English word: it can be [found in OED](https://www.oed.com/dictionary/approximatively_adv), but is obviously quite uncommon.

It is also missing from the SCWOL word lists used by `aspell`, which I see as the open source reference for spelling:
http://app.aspell.net/lookup?dict=en_US-large;words=approximatively